index.html

```diff
@@ -74,6 +74,176 @@ <h2>Let's connect</h2>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="https://cindykandie.xyz">
+                    See Project
+                </a>
+            </div>
+        </article><article class="project-card project_one">
+            <img src="assets/tonic2.svg" alt="">
+            <div class="project_info">
+                <h1>Tonic</h1>
+                <ul>
+                    <li>Canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="https://cindykandie.xyz">
+                    See Project
+                </a>
+            </div>
+        </article><article class="project-card project_one">
+            <img src="assets/multi2.svg" alt="">
+            <div class="project_info">
+                <h1>Multi-Post
+                    Stories</h1>
+                <ul>
+                    <li>Canopy</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="skills-ul">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class ="link-btn" href="#">
+                    See Project
+                </a>
+            </div>
+        </article>
+    </section>
+
+    <section class="about-section">
+        <article>
+            <h1>About <br>
+                Myself
+            </h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </article>
+        <div class="contact-me">
+            <h2>Let's connect</h2>
+            <ul class="socials">
+                <li>
+                    <a href="https://twitter.com/cindykandie">
+                        <img src="assets/twitter.svg" alt="My twitter page" >
+                    </a>
+                </li>
+               
+                <li>
```

Comment:
Good job implementing the changes requested by the previous reviewer.
I have noticed that your recent commits introduced some bugs that weren't there before, this is a media of `375px` as previously used, if you notice, you see that your page contents are overflowing to the right.

I recommend you fix this because overflowing content is a not good experience for users.


<img width="296" alt="OVERFLOW" src="https://user-images.githubusercontent.com/84930355/205431278-b2fba9b5-c32c-450d-bb6b-644347c42185.png">